app/auto-eligibility/email-notifier/index.js

```diff
@@ -15,6 +15,20 @@ const sendIneligibleApplicationEmail = async (sbi, crn, businessEmail) => {
   )
 }
 
+const sendIneligibleDefraIdRegistrationEmail = async (businessEmail) => {
```

Comment:
Can you add a unit test in for this - [index.test.js](https://github.com/DEFRA/ffc-ahwr-eligibility/blob/main/test/unit/auto-eligibility/email/index.test.js)